bin/www

```diff
@@ -25,9 +25,11 @@ var server = http.createServer(app);
  * Listen on provided port, on all network interfaces.
  */
 
-server.listen(port);
-server.on('error', onError);
-server.on('listening', onListening);
+server.listen(port, () => {
+  console.log(`The server is on http://localhost:${port}`);
```

Comment:
This is a great console log, allows to click to the localhost from your terminal!